python/ray/util/rpdb.py

```diff
@@ -43,7 +43,9 @@ def __init__(self, connection):
         self.flush = fh.flush
         self.fileno = fh.fileno
         if hasattr(fh, "encoding"):
-            self._send = lambda data: connection.sendall(data.encode(fh.encoding))
+            self._send = lambda data: connection.sendall(
+                data.encode(fh.encoding, errors="replace")
+            )
```

Comment:
The `errors='replace'` is needed since the debugger goes through `sys.stdout`, which on windows and python<3.15 will use a locale encoding. On my machine it is cp1252, which cannot represent the unicode character in `test_ray_debugger_commands`